app/v2/services/AmendFinancialDetailsService.scala

```diff
@@ -40,6 +40,7 @@ class AmendFinancialDetailsService @Inject() (connector: AmendFinancialDetailsCo
       "INVALID_EMPLOYMENT_ID"     -> NotFoundError,
       "INVALID_PAYLOAD"           -> InternalError,
       "BEFORE_TAX_YEAR_END"       -> RuleTaxYearNotEndedError,
+      "OUTSIDE_AMENDMENT_WINDOW"  -> RuleOutsideAmendmentWindow,
```

Comment:
Although it was previously added could you rename the object generally to end with Error `RuleOutsideAmendmentWindowError`

And in services with `extraTysErrors` move it into that Map